client/src/Header.js

```diff
@@ -9,7 +9,8 @@ function Header(){
     return (
     <div id = "head">
         <div id = "insta">
-        <img src = {icon} id = "icon"></img>Instaclone
+        <img src = {icon} id = "icon"></img>
+        <text id = "logo">Instaclone</text>
         </div>
         <img src = {camera} id = "camera"></img>
```

Comment:
camera icon is button here. change it to either input or button hemanth.

Reply:
okay i will
thanks for pointing it out